parsers/s01-parse/gauth-fr/navidrome-logs.yaml

```diff
@@ -0,0 +1,29 @@
+onsuccess: next_stage
+#debug: false
+name: gauth-fr/navidrome-logs
+description: "Parse navidrome logs"
+filter: "evt.Parsed.program == 'navidrome'"
+pattern_syntax:
+  NAVIDROME_CUSTOMUSER: "(%{EMAILADDRESS}|%{USERNAME})"
+  NAVIDROME_CUSTOMDATE: "%{YEAR}-%{MONTHNUM}-%{MONTHDAY}T%{HOUR}:%{MINUTE}:%{SECOND}Z"
+nodes:
+  - grok:
+      pattern: 'time="%{NAVIDROME_CUSTOMDATE:timestamp}".*msg="Unsuccessful login".*X-Forwarded-For:\[%{IP:source_ip}\].*username=%{NAVIDROME_CUSTOMUSER:username}'
```

Comment:
What if the user does not use X-Forwarded-For header?

Reply:
I won't lie i don't know :p
I guessed any reverse proxy would provide this header...
I tried thru direct IP, but in my case, these log line would not show any IP at all.
But another line would eventually show an IP. 
```time="2023-07-20T19:21:29Z" level=warning msg="HTTP: POST http://192.168.0.250:4533/auth/login" elapsedTime=1.2ms httpStatus=401 remoteAddr="172.19.0.1:52626" requestId=8c5a220df7c3/SsMOTbrPD6-018150 responseSize=40```
But in my case, since im running navidrome as a container, i just get the docker network gateway which is useless.

So any guidance is welcome actually... :)

Reply:
> I guessed any reverse proxy would provide this header...
It is best practice to pass this through, not mandatory. 

hmmm I wouldn't be happy to merge if this wasnt an option inside navidrome to set an actual trusted reverse proxy. Looking at configuration it is not possible. I will convert this to a draft and when I have more time to spend on booting up a copy and check it myself then we can explore a merge